README.md

```diff
@@ -35,7 +35,7 @@ var add = function (x, y) {
 }
 ```
 
--- YOUR ANSWER HERE --
+ES5, no, add(num1,num2)
```

Comment:
This is anonymous. The function itself has no name. This is a ```function expression``` where a function is assigned to a variable.